dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/manifest/ManifestMapper.java

```diff
@@ -129,11 +129,18 @@ private static ManifestFile mapManifestFile(ManifestFile manifestFile, Map<Strin
     return fileFields;
   }
 
+  @SuppressWarnings("unchecked")
   private static String mapValue(SearchHit hit, String fieldName) {
     val rawFieldName = IndexModel.getFileTypeModel().getField(fieldName);
-    val resultField = hit.getFields().get(rawFieldName);
+    String[] path = rawFieldName.split("\\.");
+
+    Object resultField = hit.getSource();
+    for (int i = 0; i < path.length; i++) {
```

Comment:
@btiernay , is there a more elegant way of doing this. This is still fairly neat and obvious, but is there anything clever that I'm not seeing.